internal/jimm/service_account_test.go

```diff
@@ -240,17 +241,20 @@ func TestGrantServiceAccountAccess(t *testing.T) {
 			}
 			err = pgDb.Migrate(context.Background(), false)
 			c.Assert(err, qt.IsNil)
+			groupManager, err := group.NewGroupManager(&pgDb, ofgaClient)
+			c.Assert(err, qt.IsNil)
 			jimm := &jimm.JIMM{
 				Database:      pgDb,
 				OpenFGAClient: ofgaClient,
+				GroupManager:  groupManager,
```

Comment:
i don't particularly like this, but i guess it's solved by ales's new constructor

Reply:
Ya, this be solved